client/src/components/ReviewTile.js

```diff
@@ -12,11 +15,58 @@ const ReviewTile = (props) => {
     }
```

Comment:
Ideally, we could calculate the total votes on the back end so that the front end here is responsible for displaying that information and not have to calculate this every time this component loads